pulp-api/entrypoint.sh

```diff
@@ -16,6 +19,18 @@ pulpcore-manager handle-artifact-checksums
 echo "[INFO] Collecting static files"
 pulpcore-manager collectstatic --noinput
 
+if [[ -r "$GPG_KEY" ]]; then
+echo "[INFO] Enabling Signing API"
+# Import private key
+gpg --import $GPG_KEY
+# Export public key
+gpg --export -a > /tmp/public.key
+# Export key fingerprint
+gpg /tmp/public.key 2>/dev/null | grep -E '[0-9A-Z]{40}' | awk '{print $1}' > /tmp/public.fpr
```

Comment:
Would `grep -o` (or --only-matches, or whatever the long option is) do this without the awk part?  I thought there was a way for gpg to just output the fingerprint directly.